### PR TITLE
{tools} [GCCcore/6.4.0] libssh v0.9.0 {REVIEW}

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/l/libssh/libssh-0.9.0-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+source_urls = ['https://www.libssh.org/files/0.9/']
+sources = ['%(name)s-%(version)s.tar.xz']
+checksums = ['25303c2995e663cd169fdd902bae88106f48242d7e96311d74f812023482c7a5']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('binutils', '2.28'),
+]
+
```
Comment (Micket) [inline easybuild/easyconfigs/l/libssh/libssh-0.9.0-GCCcore-6.4.0.eb]: At the very least, this probably has a OS dependency on OpenSSL?

**Review by Micket — CHANGES_REQUESTED** [review]
I'd like a second opinion on this; Do we even want libssh as a module, or do we prefer relying on OS to keep it in regards to security updates? Perhaps it's enough to keep openssl as OS level.

**Comment by akesandgren** [comment]
I'm for keeping things like this out of EB and relying on OS builds due to the frequent security updates.

**Comment by boegel** [comment]
We have easyconfig files for `OpenSSL` too, but we actively discourage using them to resolve dependencies.

I think it's OK to add an easyconfig for this, it could help people that don't have admin access to install a missing dep?

File: easybuild/easyconfigs/l/libssh/libssh-0.9.0-GCCcore-6.4.0.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/l/libssh/libssh-0.9.0-GCCcore-6.4.0.eb]: So, this should either add a dep. on OpenSSL or add a os dependency on it.

Comment (smoors) [reply easybuild/easyconfigs/l/libssh/libssh-0.9.0-GCCcore-6.4.0.eb]: @SimonPinches please add an os dependency for SSL, as @Micket requested?
```
osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

Comment (SimonPinches) [reply easybuild/easyconfigs/l/libssh/libssh-0.9.0-GCCcore-6.4.0.eb]: Done - apologies for the delay!

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node353.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/59156ab53d1f20c1219edfdc463c5cf5 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Review by Micket — APPROVED** [review]

**Comment by smoors** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**